aerleon/lib/nokiasrl.py

```diff
@@ -81,7 +113,38 @@ def SetOptions(self) -> None:
             )
         if 'not-syn-ack' in opts:
             self.term_dict['match']['tcp-flags'] = "!(syn&ack)"
-        # Note: not handling established | tcp-established, could throw error
+
+        def _tcp_established():
+            self.term_dict['match']['tcp-flags'] = "ack|rst"
```

Comment:
This seems wrong to me, wouldn't tcp established be syn and ack?

Reply:
See https://github.com/aerleon/aerleon/blob/main/aerleon/lib/juniper.py#L182

Based on customer feedback